### PR TITLE
Added version for scala211 which somehow got lost during update of the

### PR DESCRIPTION
**Pull request by pebra**
Hey, I think there is now way to get scala 2.11 from brew anymore, so this PR attempts to fix that.
I hope this is correct, at least brew audit didn't return any errors.

- [] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md) document?
- [] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-versions/pulls) for the same formula update/change?
- [] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?
- [] Does your submission adhere to Versions "Acceptable Formulae" [guidelines](https://github.com/Homebrew/homebrew-versions/blob/master/README.md#acceptable-formulae)?

-----

scala formula(2.11)

### CODE REVIEW

File: scala211.rb [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  url "http://downloads.lightbend.com/scala/2.11.8/scala-2.11.8.tgz"
+  sha256 "87fc86a19d9725edb5fd9866c5ee9424cdb2cd86b767f1bb7d47313e8e391ace"
+
+  bottle do
+    cellar :any_skip_relocation
+    rebuild 1
+    sha256 "2021f0815655cd419fa5ef3fec5f57b6181943ec57af8a1bee68976f2872d942" => :sierra
+    sha256 "2021f0815655cd419fa5ef3fec5f57b6181943ec57af8a1bee68976f2872d942" => :el_capitan
+    sha256 "2021f0815655cd419fa5ef3fec5f57b6181943ec57af8a1bee68976f2872d942" => :yosemite
+  end
```
Comment (MikeMcQuaid) [inline scala211.rb]: Remove this bottle block.

**Comment by MikeMcQuaid** [comment]
@pebra Thanks for the PR! Please also remove `scala210.rb` in this PR so we only have two versions here. Thanks!

**Comment by pebra** [comment]
Hey, thanks for replying so quickly! Removed the bottle block and removed the formula for scala 2.10

**Comment by pebra** [comment]
@MikeMcQuaid  I think you meant to remove scala 2.9 right?

**Comment by MikeMcQuaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format is `scala211 2.11.8 (new formula)`. Please complete the issue template in future PRs where this format is detailed in the linked [Contributing guidelines](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md). Thanks!

**Pull request closed without merging**